README.md

```diff
@@ -212,10 +212,29 @@ console.log(
 ```
 
 
-## CommonJS modules
+## Others
+
+### Write code in CommonJS modules
 
 If you want to use CommonJS modules, `require` the code from `@puppeteer/replay/cjs`. For example,
 
 ```js
 const { stringify } = require('@puppeteer/replay/cjs');
 ```
+
+### Create a Chrome extension for Recorder
+
+You can create a Chrome extension for [Recorder](https://goo.gle/devtools-recorder). Refer to the [Chrome Extensions documentation](https://developer.chrome.com/docs/extensions/mv3/devtools/) for more details on how to extending DevTools.
+
+This repository contains an [example extension](https://github.com/puppeteer/replay/tree/main/examples/chrome-extension). Once installed, the Recorder will have a new export option **Export as a Custom JSON script** in the [export dropdown](https://developer.chrome.com/docs/devtools/recorder/#export-flows).
+
+To load the example into Chrome DevTools. Follow these steps:-
+
+1. Download the [chrome-extension](https://github.com/puppeteer/replay/tree/main/examples/chrome-extension) folder.
+2. [Load the folder as unpacked extension](https://developer.chrome.com/docs/extensions/mv3/getstarted/#unpacked) in Chrome. 
+3. [Open a recording](https://developer.chrome.com/docs/devtools/recorder/#record) in the Recorder.
+4. Click on [export](https://developer.chrome.com/docs/devtools/recorder/#export-flows). A new **Export as a Custom JSON script** menu is added.
```

Comment:
`A new **Export as a Custom JSON script** menu is added` sentence feels a bit out of the flow. Maybe we can say `Now you can see a new **Export as a Custom JSON script** action in the export menu`. WDYT?